project/datasets/modeled/dgen/dimensions/sectors.csv

```diff
@@ -0,0 +1,3 @@
+id,name
+res,Residential
+com,Commercial
```

Comment:
I was expecting res and non-res. Either way is fine, just be consistent here and in dataset.json5 and in the data files.